bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/WZAToLNuLLA_4f_NLO/WZAToLNuLLA_4f_NLO_run_card.dat

```diff
@@ -0,0 +1,145 @@
+ 1        = QES_over_ref     ! ratio of current QES over reference QES
+#*********************************************************************** 
+# Reweight flags to get scale dependence and PDF uncertainty           *
+# For scale dependence: factor rw_scale_up/down around central scale   *
+# For PDF uncertainty: use LHAPDF with supported set                   *
+#***********************************************************************
+ .true.   = reweight_scale   ! reweight to get scale dependence
+  0.5     = rw_Rscale_down   ! lower bound for ren scale variations
+  2.0     = rw_Rscale_up     ! upper bound for ren scale variations
+  0.5     = rw_Fscale_down   ! lower bound for fact scale variations
+  2.0     = rw_Fscale_up     ! upper bound for fact scale variations
+#***********************************************************************
+# Merging - WARNING! Applies merging only at the hard-event level.     *
+# After showering an MLM-type merging should be applied as well.       *
+# See http://amcatnlo.cern.ch/FxFx_merging.htm for more details.       *
+#***********************************************************************
+ 0        = ickkw            ! 0 no merging, 3 FxFx merging
+#***********************************************************************
+#
+#***********************************************************************
+# BW cutoff (M+/-bwcutoff*Gamma)                                       *
+#***********************************************************************
+ 30  = bwcutoff
+#***********************************************************************
+# Cuts on the jets                                                     *
+# Jet clustering is performed by FastJet.
+# When matching to a parton shower, these generation cuts should be    *
+# considerably softer than the analysis cuts.                          *
+# (more specific cuts can be specified in SubProcesses/cuts.f)         *
+#***********************************************************************
+   1  = jetalgo   ! FastJet jet algorithm (1=kT, 0=C/A, -1=anti-kT)
+ 0.7  = jetradius ! The radius parameter for the jet algorithm
+  15  = ptj       ! Min jet transverse momentum
+  -1  = etaj      ! Max jet abs(pseudo-rap) (a value .lt.0 means no cut)
+#***********************************************************************
+# Cuts on the charged leptons (e+, e-, mu+, mu-, tau+ and tau-)        *
+# (more specific gen cuts can be specified in SubProcesses/cuts.f)     *
+#***********************************************************************
+   5  = ptl     ! Min lepton transverse momentum
+  5.0 = etal    ! Max lepton abs(pseudo-rap) (a value .lt.0 means no cut)
+   0  = drll    ! Min distance between opposite sign lepton pairs
+   0  = drll_sf ! Min distance between opp. sign same-flavor lepton pairs
+   1  = mll     ! Min inv. mass of all opposite sign lepton pairs
+  20  = mll_sf  ! Min inv. mass of all opp. sign same-flavor lepton pairs
```

Comment:
Just to understand. Why did you pick 20 GeV ?

Reply:
In our testing, a small mll_sf value will cause some problems.
e.g.
If we generate the samples with w+ and w- seperately, the cross section doesn't match well , i.e. XS(w+)+XS(w-) has a obvious discrepancy with XS(w). This problem is solved after we adding up this value so we finally choose 20.

Reply:
> In our testing, a small mll_sf value will cause some problems.
> e.g.
> If we generate the samples with w+ and w- seperately, the cross section doesn't match well , i.e. XS(w+)+XS(w-) has a obvious discrepancy with XS(w). This problem is solved after we adding up this value so we finally choose 20.

Can you tell us how big the difference was? This is interesting...

Reply:
e.g. for mll_sf=1, after generating the gridpack, we use ./runcmsgrid.sh to generate 1K events for testing. Since in MG NLO production, the cross section isn't related to the event number so 1K is enough for testing.
XS(W) = 1.961e-02 +- 2.0e-05 pb
XS(W+) = 7.896e-03 +- 8.3e-06 pb
XS(W-) = 5.387e-03 +- 5.5e-06 pb
So you can see the discrepancy.

If we use mll_sf=20, we can get
XS(W) = 1.879e-02 +- 1.9e-05 pb
XS(W+) = 1.124e-02 +- 1.2e-05 pb
XS(W-) = 7.591e-03 +- 7.9e-05 pb
Though the cross section is smaller due to the tighter cut, the discrepancy is reduced in this case.

Reply:
Particles that decays using MadSpin won't get the cuts applied. You can check it with simpler process like "p p > w+" and doing the decays with MadSpin "w+ > e+ ve". Compare the cross sections before and after giving the lepton pt cuts, the numbers will be same.
So mll_sf cut only affects the 2 leptons that you make in NLO that comes from Z/gamma. Now if "mll_sf=1 and mll=1", these two are doing exactly the same thing, cut your dilepton masses to be above 1 GeV. "mll_sf=20 and mll=1" will give a cut to dilepton masses at 20GeV since mll=1 will be overwritten.
Because of the gamma divergence "mll~0GeV", the computation might not be stable. Can you use 10 GeV cut instead of what you are using now (mll_sf=20)? This is typical number that is used to avoid gamma divergence problems.
mll=10 or mll_sf=10 either will give the same cuts to your gridpacks since two leptons are always same flavour.

Reply:
Thanks a lot for your comments! Yes in this case mll_sf is actually the same as mll, so I keep the mll at 1 GeV.
I will change the mll_sf from 20 to 10 GeV.